catalog/spatial/wfs/1.1.0/spatial-wfs-v1_1_0-source/src/main/java/org/codice/ddf/spatial/ogc/wfs/v110/catalog/source/WfsSource.java

```diff
@@ -239,51 +238,18 @@
   }
 
   public WfsSource(
-      FilterAdapter filterAdapter,
-      BundleContext context,
-      AvailabilityTask task,
       ClientFactoryFactory clientFactoryFactory,
       EncryptionService encryptionService,
-      WfsMetacardTypeRegistry wfsMetacardTypeRegistry,
-      List<MetacardTypeEnhancer> metacardTypeEnhancers)
-      throws SecurityServiceException {
-
-    this.filterAdapter = filterAdapter;
-    this.context = context;
-    this.availabilityTask = task;
+      ScheduledExecutorService scheduler) {
     this.clientFactoryFactory = clientFactoryFactory;
     this.encryptionService = encryptionService;
-    this.wfsMetacardTypeRegistry = wfsMetacardTypeRegistry;
-    this.metacardTypeEnhancers = metacardTypeEnhancers;
-    this.wfsMetadata =
-        new WfsMetadataImpl<>(
-            this::getId,
-            this::getCoordinateOrder,
-            Collections.singletonList(FEATURE_MEMBER_ELEMENT),
-            FeatureTypeType.class);
-    initProviders();
-    createClientFactory();
-    configureWfsFeatures();
-  }
```

Comment:
❓ Looks like we took some field initialization logic out of the constructor, but didn't take the fields out of the class entirely, for instance: `filterAdapter` and `context`.

How are these fields supposed to being instantiated now?

Reply:
They're injected through the setters via the `<property>` tags in blueprint.